openfisca_aotearoa/variables/acts/social_security/disability_allowance/disability_allowance.py

```diff
@@ -0,0 +1,220 @@
+        meets_clause_11_conditions = persons("disability_allowance__income_limit_clause_11", period)  # single person w/out dep children
+        meets_clause_12_conditions = persons("disability_allowance__income_limit_clause_12", period)  # in relationship
+        meets_clause_13_conditions = persons("disability_allowance__income_limit_clause_13", period)  # single person with 1 child
+        meets_clause_14_conditions = persons("disability_allowance__income_limit_clause_14", period)  # single person with > 1 child
+
+        return meets_clause_10_conditions + meets_clause_11_conditions + meets_clause_12_conditions + \
+            meets_clause_13_conditions + meets_clause_14_conditions
+
+
+class disability_allowance__ongoing_additional_expenses(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person have additional expenses of an ongoing kind arising from the person’s disability?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__income_limit_clause_10(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 10?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        person_aged_16_or_17 = (persons("age", period.start) >= 16) * (persons("age", period.start) <= 17)
+        no_partners = numpy.logical_not(persons("social_security__in_a_relationship", period))
+        income_within_limit = persons("social_security__income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_10"]
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
+        return person_aged_16_or_17 * no_partners * income_within_limit * without_dependant_child
+
+
+class disability_allowance__income_limit_clause_11(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 11?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        person_not_aged_16_or_17 = (persons("age", period.start) < 16) + (persons("age", period.start) > 17)
+        no_partners = numpy.logical_not(persons("social_security__in_a_relationship", period))
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
```

Comment:
This is just going to check if each person is dependent child. Try maybe:

```suggestion
        dependent_children = persons("social_security__dependent_children", period)
        without_dependent_child = persons.has_role(entities.Family.PRINCIPAL) * (persons.family.sum(dependent_child, role = entities.Family.CHILD) == 0)
```